### ISSUE TITLE
Verifying the write to head LEDs consumes serial use

### ISSUE THREAD

**Issue by Tiogaplanet** [adamgreen/MiP_ProMini-Pack#15]
More of a question/suggestion than actual issue.  For animating the eyes, the verification that the write was successful uses the serial line unnecessarily, IMO.  Any appetite or suggestions for disabling the check?  Maybe disabling the checks or making two new members called: 

```
void writeHeadLEDs(MiPHeadLED led1, MiPHeadLED led2, MiPHeadLED led3, MiPHeadLED led4, boolean doCheck);
void writeHeadLEDs(const MiPHeadLEDs& headLEDs, boolean doCheck);
```

For example, I am doing writes of random on/off values to each of the four eye LEDs every 500ms to indicate rain.  I don't need to verify that every random write was successful.

**Comment by adamgreen** [comment]
I like the idea of adding new methods which skip the verification process for setting LEDs since as you mention there might be times when the user doesn't care if they make it through since they will set them again soon anyway. I would prefer to keep the default as it is where if you ask the library to change the LED setting, it does its best to make sure that the LED is switched to the requested state. This is more consistent with the rest of the library and hides the ugliness of error detection and recovery from the Arduino end user.

My current thought on the names of the new functions would be something like:
```c++
void unverifiedWriteHeadLEDs();
void unverifiedWriteChestLED();
```
I thought about other prefixes like _attempt_, _unsafe_, _lossy_, _unchecked_, etc. but I think _unverified_ works best. What do you think?

**Comment by Tiogaplanet** [comment]
I like the use of `unverified` as a descriptor.  

Next question for you, since the library has three declarations for `writeChestLED()` and two for `writeHeadLEDs()` should there be an unverified form of each one?  I am thinking not because a user may not be doing quick writes in succession when using the onTime and offTime variables.  So, I think there should be just

```
void unverifiedWriteChestLED(uint8_t red, uint8_t green, uint8_t blue);
void unverifiedWriteHeadLEDs(MiPHeadLED led1, MiPHeadLED led2, MiPHeadLED led3, MiPHeadLED led4);
void unverifiedWriteHeadLEDs(const MiPHeadLEDs& headLEDs);
```

Agree, disagree?

**Comment by adamgreen** [comment]
I am inclined to do it for all of them. I can think of some scenarios where a developer might want to update the LED settings, including on/off times, at regular intervals. One example is if a distance sensor (such as ultrasonic) was connected to the MiP ProMini and they wanted to set the blink rate based on the latest ultrasonic reading. If one of those LED updates doesn't make it through, it wouldn't be the end of the world.

**Comment by adamgreen** [comment]
Fixed with commit 065181cd229cabad0b67e8999c228b052c2bd145
